custom_components/tuya_local/devices/blitzwolf_bwsh5_humidifier.yaml

```diff
@@ -29,7 +29,7 @@ primary_entity:
     - id: 13             
       name: humidity
       type: integer
-      optional: true
+      #optional: true
```

Comment:
I think this was made optional based on previous logs that showed it missing.
Removing that now will affect compatibility, as the device appears to be inconsistent about which dps it reports (probably based on the mode it is in).